src/components/QuizTemplate.tsx

```diff
@@ -132,9 +133,31 @@ const QuizTemplate: React.FC<QuizProps> = QuizProps => {
     return shuffleModalArr[0];
   };
 
-  const checkAnswer = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+  const selectOption = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    // Set the selected option
+    setSelectedOption(e.currentTarget.value);
+
+    // Get answer buttons
+    const answerBtns = document.getElementsByClassName("answers-btns")
```

Comment:
In React, you want to avoid directly manipulating the DOM when possible.
A better option would be to use refs instead

Here are the updates you will need to add to make this more idiomatic in react

1. import `useRef` at the top of file `import React, { useState, useEffect, useRef } from "react";`
2. add this const variable at the top too `const answerButtonsRef = useRef(null);`
3. then replace line 141 and use the ref instead ` const answerBtns = answerButtonsRef.current.getElementsByClassName("answers-btns");`
4. inside the question props object, add the `answerButtonsRef`
5. Inside the Questions.tsx file, add the `answerButtonsRef: React.MutableRefObject<any>;` to the QuizProps
6. then add the new prop inside the ul element `<ul ref={QuizProps.answerButtonsRef}>`

Reply:
There's an issue with eslint for your suggestion on line 5. It says no explicit any.